src/SearchByTitleForm/SearchByTitleForm.js

```diff
@@ -1,34 +1,42 @@
 import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
 import '../Navbar/Navbar.css';
-import { withRouter } from 'react-router';
 
 class SearchByNameForm extends Component {
-    constructor(props) {
-        super(props);
+    constructor() {
+        super();
         this.state = {
             movieTitle: '',
+            movieID: '',
+            submit: false,
```

Comment:
This new `submit` value controls the Redirect. When a movie title to be searched is submitted, the value is switched to `true`. Then when the component updates again, this value is switched back to `false`

Reply:
interesting! I might need you to walk me through some of this later so i understand it better if you dont mind :)